decidim-meetings/app/helpers/decidim/meetings/admin/application_helper.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Meetings
+    module Admin
+      # Custom helpers, scoped to the meetings admin engine.
+      #
+      module ApplicationHelper
+        include Decidim::MapHelper
+
+        def meeting_organizer_picker_text(form)
+          return "" unless form.object.organizer.present?
+          return "#{form.object.organizer.name} (@#{form.object.organizer.nickname})" if form.object.organizer.present?
```

Comment:
Ne need for that second `return`, and the `if` part is redundant as it's handled in the previous line